src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.LeadingZeroCount.cs

```diff
@@ -41,6 +42,30 @@ public static Vector128<T> Invoke(Vector128<T> x)
                 {
                     if (sizeof(T) == 4) return Avx512CD.VL.LeadingZeroCount(x.AsUInt32()).As<uint, T>();
                     if (sizeof(T) == 8) return Avx512CD.VL.LeadingZeroCount(x.AsUInt64()).As<ulong, T>();
+                    if (sizeof(T) == 2)
+                    {
+                        Vector128<uint> lowHalf = Vector128.Create((uint)0x0000FFFF);
+                        Vector128<uint> x_bot16 = Sse2.Or(Sse2.ShiftLeftLogical(x.AsUInt32(), 16), lowHalf);
+                        Vector128<uint> x_top16 = Sse2.Or(x.AsUInt32(), lowHalf);
+                        Vector128<uint> lz_bot16 = Avx512CD.VL.LeadingZeroCount(x_bot16);
+                        Vector128<uint> lz_top16 = Avx512CD.VL.LeadingZeroCount(x_top16);
+                        Vector128<uint> lz_top16_shift = Sse2.ShiftLeftLogical(lz_top16, 16);
+                        return Sse2.Or(lz_bot16, lz_top16_shift).AsUInt16().As<ushort, T>();
+                    }
+                }
+                if (Avx512Vbmi.VL.IsSupported && sizeof(T) == 1)
```

Comment:
Nit: can you please:
- order these if statements by sizeof(T)
- change the existing one liners to be blocks now that there are others with blocks
- put a blank line between each if block